views/_includes/errorList.pug

```diff
@@ -0,0 +1,6 @@
+div.error-list(role='alert')
+    h3.error-list__header #{__('Please correct the errors on the page')}
+    ol#formErrors.error-list__list
+        each error in errors
+            li.error-list__list-item
+                a(class='error-list__link' id=error.param+'_error' href='#'+error.param) #{error.msg}
```

Comment:
I've used BEM in the past too! I haven't been doing anything specific in this repo so far but if you want to continue with BEM CSS conventions, I'm game 👍